src/interaction/inspector.jl

```diff
@@ -815,13 +815,18 @@ function show_data(inspector::DataInspector, plot::BarPlot, idx)
     tt = inspector.plot
     scene = parent_scene(plot)
 
-    pos = plot[1][][idx]
-    proj_pos = shift_project(scene, apply_transform_and_model(plot, to_ndim(Point3f, pos, 0)))
+    bbox = plot.plots[1][1][][idx]
+    tt_frac = if plot.direction[] === :y
+        Point2f(0.5, 1.0)
+    else
+        Point2f(1.0, 0.5)
+    end
+    tt_pos = fractionpoint(bbox, tt_frac)
+    proj_pos = shift_project(scene, apply_transform_and_model(plot, to_ndim(Point3f, tt_pos, 0)))
```

Comment:
```suggestion
    proj_pos = Point2f(mouseposition_px(inspector.root))
```
to switch to a tooltip anchored to the mouse position